Sources/NIOMulticastChat/main.swift

```diff
@@ -11,6 +11,7 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 //===----------------------------------------------------------------------===//
+import CNIOLinux
```

Comment:
This seems like an odd one to need in an example. What symbol is being complained about?

Reply:
`addr.address.sin_addr`
```
 94 |             return provider.setIPMulticastIF(addr.address.sin_addr).map { channel }
    |                                                           `- error: property 'sin_addr' is not available due to missing import of defining module 'CNIOLinux'
 ```